src/schema/v2/recentlySoldArtworks.ts

```diff
@@ -0,0 +1,173 @@
+import { GraphQLFieldConfig, GraphQLObjectType, GraphQLString } from "graphql"
+import { connectionDefinitions, connectionFromArray } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { createPageCursors } from "schema/v1/fields/pagination"
+import { ResolverContext } from "types/graphql"
+import { ArtworkType } from "./artwork"
+
+const RecentlySoldArtworkType = new GraphQLObjectType<any, ResolverContext>({
+  name: "RecentlySoldArtworkType",
+  fields: {
+    lowEstimateUSD: {
+      type: GraphQLString,
```

Comment:
For pricing fields, rather than returning a pre-formatted string, I'd encourage us to move towards the [`Money` type](https://github.com/artsy/metaphysics/blob/eb885d10dd0f8ac101e7c8bc4bc20e58055b38dd/src/schema/v2/fields/money.ts#L158-L189) which implements good practices around currencies, supports formatting by clients, etc.